std/random.d

```diff
@@ -1349,39 +1482,17 @@ if (isUnsigned!UIntType)
     {
         return this;
     }
-
-
-    /**
-     * Compares against $(D_PARAM rhs) for equality.
-     */
-    bool opEquals(ref const XorshiftEngine rhs) const @safe pure nothrow @nogc
```

Comment:
This struct's custom `opEquals` appeared to be the same as default struct equality so I removed it.